src/pages/PersonProfile/components/HireForm.jsx

```diff
@@ -1,10 +1,16 @@
-import { useState } from 'react'
-
-function HireForm(props) {
-  const [wage, setWage] = useState(0)
-
+/* eslint-disable react/prop-types */
+import { useState } from "react";
+import { Link, useNavigate } from "react-router-dom";
+function HireForm({ person, setHiredPeople, hiredPeople }) {
+  const [wage, setWage] = useState(0);
+  const navigate = useNavigate();
   function handleSubmit(event) {
-    event.preventDefault()
+    event.preventDefault();
+    person.wage = wage;
```

Comment:
you should create a new person object to ensure react knows the object is different

```js
const updatedPerson = {...person, wage}
```